play/PlayerRules/CharacterCreation/Class-Specific Documentation/Power Suits/Chassis.txt

```diff
@@ -0,0 +1,19 @@
+		Light Chassis
```

Comment:
/shrug?
No idea! Want to use "light" chassis to lend their mobility to the creation of disruption tanks, medium chassis maybe have more slots which will tend toward making damage tanks and heavy chassis easier to armor up to make an anchor tank?

or we could thow the class distinctions out of the window and have each chassis be unique unto themselves. I have no preference.